doc/lightning-createonion.7.md

```diff
@@ -132,5 +132,5 @@ RESOURCES
 
 Main web site: <https://github.com/ElementsProject/lightning>
 
-[bolt04]: https://github.com/lightningnetwork/lightning-rfc/blob/master/04-onion-routing.md
+[bolt04]: https://github.com/lightning/bolts/blob/master/04-onion-routing.md
```

Comment:
nit: It is not broken the `https://github.com/lightningnetwork/lightning-rfc/blob/master/04-onion-routing.md` 
 will resolve in `https://github.com/lightning/bolts/blob/master/04-onion-routing.md` is only a redirection due the spec rebranding

Reply:
@vincenzopalazzo are you sure?

![Screenshot from 2022-06-29 19-45-38](https://user-images.githubusercontent.com/1189245/176439647-17ae316e-2032-43e4-b4d8-ef667176ff88.png)

Reply:
Something weird is going on here. The link is clearly broken, but it could be due to static site generator stripping `.md` or something

Reply:
yes, it is wrong the link that you click in some way! but  if you click [here](https://github.com/lightningnetwork/lightning-rfc/blob/master/04-onion-routing) you will be redirected correctly.

Maybe you can use the []() format instead the current format with label

Reply:
> but if you click [here](https://github.com/lightningnetwork/lightning-rfc/blob/master/04-onion-routing) you will be redirected correctly

nope, it's 404 =)

Reply:
In fact, most of the links in the doc are currently broken